src/components/OcCheckbox.spec.js

```diff
@@ -0,0 +1,66 @@
+import OcCheckbox from "./OcCheckbox.vue"
+import { mount } from "@vue/test-utils"
+
+describe("OcCheckbox", () => {
+  function getWrapperWithProps(props) {
+    return mount(OcCheckbox, {
+      propsData: {
+        label: "test label",
+        ...props,
+      },
+    })
+  }
+  const checkboxSelector = "input[type='checkbox']"
+
+  describe("input id", () => {
+    it("should set the provided id for the input", () => {
+      const wrapper = getWrapperWithProps({ id: "test-id" })
+      const checkBoxElement = wrapper.find(checkboxSelector)
+      expect(checkBoxElement.exists()).toBeTruthy()
+      expect(checkBoxElement.attributes("id")).toBe("test-id")
+    })
+  })
+  describe("input label", () => {
+    it("should set the provided label for the input", () => {
+      const wrapper = getWrapperWithProps({ id: "test-id", label: "test label" })
```

Comment:
you can remove `label` from here as it is same as the default value
```suggestion
      const wrapper = getWrapperWithProps({ id: "test-id" })
```